Randolph-dev/Druid_Partnering_Project_React24k
# Scrum 90 style the jobs page

## Related ticket:
[SCRUM-90](https://druidpartneringproject2024bch.atlassian.net/browse/SCRUM-90?atlOrigin=eyJpIjoiYTQxYzZkMmI3Y2QyNGQ4ZDk3NDQzODQzMWVkNzQ0MzkiLCJwIjoiaiJ9)

## In this PR:
- Render the data
- Add basic styling


## Testing instruction
- Checkout branch
- Make sure Drupal is running, imported db and configuration is the latest
- Go to the Jobs page
- Checkout the page


[SCRUM-90]: https://druidpartneringproject2024bch.atlassian.net/browse/SCRUM-90?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
